package/MDAnalysis/topology/GROParser.pyx

```diff
@@ -84,6 +85,16 @@ def parse(self, **kwargs):
         """Return the *Topology* object for this file"""
         # Gro has the following columns
         # resid, resname, name, index, (x,y,z)
+        cdef int n_atoms;
+        cdef int i;
+        cdef str line
+        cdef int count = 0
+        cdef int[:] resids
+        cdef object[:] resnames
+        cdef object[:] names
+        cdef int[:] indices
+        cdef long long[:] starts
```

Comment:
pretty sure we just need ints here too

Reply:
Travis was failing with `int` and expecting it to be `long long` as I remember. I'll double check.

Reply:
`np.where` returns a np.int64 but mac integer is 32-bit as the same issure [here](https://stackoverflow.com/questions/32262976/cython-buffer-type-mismatch-expected-int-but-got-long), I've type-casted the arrays inplace with `np.int32` and works with `int` now.